web_src/js/index.js

```diff
@@ -743,6 +743,17 @@ async function initRepository() {
         if (typeof $(this).data('context') !== 'undefined') $($(this).data('context')).addClass('disabled');
       }
     });
+
+    // Enable number of webhooks to keep.
```

Comment:
as comment above - do we realy have to set it on repo creation!
I think it get to overloaded

Reply:
this is for the edit repo page where you could change the settings, though I guess if the design was changed as I think you suggested the page would need to work a little different than before

Reply:
U let me have a look ...

Reply:
ok you can resolve this one :)